Controller/Controller.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="5.0.0" />
     <PackageReference Include="NAudio" Version="1.10.0" />
     <PackageReference Include="SSH.NET" Version="2016.1.0" />
+    <PackageReference Include="taglib" Version="2.1.0" />

```

Comment:
Waarvoor gebruiken we deze library?

Reply:
voor het uitlezen van de ID3 tags van de songs. dingen als naam, duur en later misschien nog album ofzo

Reply:
Maar wat is een ID3 tag überhaupt?? :P

Reply:
ID3 is a metadata container most often used in conjunction with the MP3 audio file format. It allows information such as the title, artist, album, track number, and other information about the file to be stored in the file itself.